### PR TITLE
New homepage section backgrounds

### PR DESCRIPTION
**Pull request by keeeeeegan**
Updates the backgrounds on the stats, ignite, and event sections on mobile.

### CODE REVIEW

File: dev/custom-pages/homepage.scss [hunk 1]
```diff
@@ -896,13 +916,22 @@ $color-linkedin: #0077b5;
...
-      background-image: none !important;
+      background: $black url($events-background-mobile-url) !important;
+      background-position: 50% 75% !important;
+      background-repeat: no-repeat !important;
+      background-size: cover !important;
+      padding: 30px 0 300px;
+    }
+
+    @media (max-width: $screen-xs-max) {
+      padding: 5px 0 220px;
```
Comment (cjg89) [inline dev/custom-pages/homepage.scss]: You may want to add a separate breakpoint that increases padding-bottom for ~480px and lower; with 220px bottom padding I'm seeing some text overflow into the background in QA.

The mobile background image also has a lot of plain black empty space that could be cropped out--if the image were cropped without the empty space, you could do something like:
```
background-position: center bottom;
padding-bottom: <some value in vw units that corresponds to the aspect ratio of the image>
```
to ensure the padding always matches the image's dimensions.

**Comment by keeeeeegan** [comment]
Updated the padding using vw units.

File: dev/custom-pages/homepage.scss [hunk 2]
```diff
@@ -924,14 +925,15 @@ $color-linkedin: #0077b5;
...
     @media (max-width: $screen-sm-max) {
       background: $black url($events-background-mobile-url) !important;
-      background-position: 50% 75% !important;
+      background-position: center bottom !important;
       background-repeat: no-repeat !important;
-      background-size: cover !important;
-      padding: 30px 0 300px;
+      background-size: contain !important;
+      padding: 30px 0;
+      padding-bottom: 49vw;
```
Comment (cjg89) [inline dev/custom-pages/homepage.scss]: Padding-bottom is defined twice

File: dev/custom-pages/homepage.scss [hunk 3]
```diff
@@ -521,16 +521,17 @@ $color-linkedin: #0077b5;
...
     @media (max-width: $screen-sm-max) {
       background: $black url($ignite-background-mobile-url) !important;
-      background-position: 50% 75% !important;
+      background-position: center bottom !important;
       background-repeat: no-repeat !important;
-      background-size: cover !important;
-      padding: 30px 0 300px;
+      background-size: contain !important;
+      padding: 30px 0;
+      padding-bottom: 62vw;
```
Comment (cjg89) [inline dev/custom-pages/homepage.scss]: Padding-bottom is defined twice

**Comment by keeeeeegan** [comment]
Updated with new single-line padding declarations.

**Pull request merged**